### DISCUSSION TITLE
Date and time formats

### DISCUSSION THREAD

**Discussion by ben-biber** [pretix/pretix#3484] [General]
How do I change the  Date and time format so it doesn't look like this:

Beginn: 2 nachm.
Ende: 3 nachm.

**Comment by raphaelm** [comment]
I have never in nine years even once seen pretix to say "2 nachm.". Where does pretix do that?

**Reply by ben-biber** [reply]
Here from my installation: https://pretix.kjg-dossenheim.org/kjg-dossenheim/freizeit-2023/

**Reply by wiffbi** [reply]
That looks strange, as if de-localization uses a en_US dateformat (that is the only place where we use am/pm formatting for `TIME_FORMAT`).

Probably unrelated, but there seems to be something broken with cloudflare and our JavaScript as the lightbox is embedded twice and thus not hidden as only the first lightbox is selected due to reuse of the same id-attribute.

**Reply by ben-biber** [reply]
<img width="1068" alt="image" src="https://github.com/pretix/pretix/assets/24373331/b0262b3e-6d74-4c0a-abdd-288d305702d2">
Weirdly enough the region setting is in English but the rest in German
